integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/commands/AbstractRequestCmd.java

```diff
@@ -417,10 +422,14 @@ public CommandResult process(CommandInvocation commandInvocation) throws Command
                     } else {
                         printAsCsv(rootNode, returnFields, unquoted);
                     }
-                } catch (Exception ignored) {
-                    copyStream(new ByteArrayInputStream(buffer.toByteArray()), abos);
+                } catch (Exception e) {
+                    throw new RuntimeException("Error processing results: " + e.getMessage(), e);
                 }
             } else {
+                if (outputFormat != OutputFormat.JSON || returnFields != null) {
```

Comment:
```suggestion
                if (!pretty && (outputFormat != OutputFormat.JSON || returnFields != null)) {
```

Reply:
@vmuzikar !pretty means compressed.  Do you only want to check if you can't apply the format / returned fields if compressed?

Reply:
Sorry, should've been probably:
```suggestion
                if (pretty && (outputFormat != OutputFormat.JSON || returnFields != null)) {
```
Meaning we don't print the error message if compressed.

Reply:
`--compressed` is a user defined option which is not compatible with either `--format` or `--fields`. I think we don't need to print an error and just make `--compressed` to take precedence over the other fields. This is also aligned with the previous behaviour, otherwise we're making a breaking change basically.

However, if we want to print an error, IMHO we should make the error message more clear in case it's caused by `--compressed`.

Reply:
> This is also aligned with the previous behaviour, otherwise we're making a breaking change basically.

With respect to a breaking change the previous behavior would always produce a result, possibly silently ignoring if it could not apply the format or fields.  Do you want to retain that behavior?  Or do you want to want to expand on the message to mention the either it's compressed or not json?

Reply:
Either way works for me.

But I think I'm more inclined towards going with the error message, it feels cleaner to me. But we probably should mention that in upgrading guide.

WDYT?